pkgs/top-level/perl-packages.nix

```diff
@@ -19969,6 +19969,8 @@ with self; {
     };
   };
 
+  ParseWin32Registry = callPackage ../development/perl-modules/ParseWin32Registry { };
```

Comment:
I recommend defining `perlPackages.ParseWin32Registry` similar to how other packages are defined in `perl-packages.nix` unless there is a specific reason for it being defined in its own .nix file.

When it is defined in perl-packages.nix, it will also be updated semi-automatically when we do mass updates.

Reply:
Also, 
- The commit message `perl.pkgs.ParseWin32Registry: init at 1.1` should be `perlPackages.ParseWin32Registry: init at 1.1`
- The body of the commit messages contain multiple lines like "Update perl-packages.nix" and "Update package.nix" that are not relevant, please reword these.
- `perl` is not a required buildInput for `buildPerlPackage`

Reply:
@stigtsp if I do the asked change in `perl-packages.nix` as you suggested, should I remove the file `pkgs/development/perl-modules/ParseWin32Registry/default.nix` ?

Reply:
> @stigtsp if I do the asked change in `perl-packages.nix` as you suggested, should I remove the file `pkgs/development/perl-modules/ParseWin32Registry/default.nix` since their content will be very similar ?

Yes please. Parse::Win32Registry seems like a straight forward module without any non-core dependencies.

Reply:
Fixed as requested @stigtsp

Reply:
@stigtsp fixed. Can you give a look if everythng is right?

Reply:
Should be good now, thanks! 👍